modules/tracktion_engine/model/export/tracktion_Renderer.cpp

```diff
@@ -446,8 +447,16 @@ bool Renderer::renderToFile (const juce::String& taskDescription,
             }
             else
             {
-                while (task->runJob() == juce::ThreadPoolJob::jobNeedsRunningAgain)
-                {}
+                while (true) {
+                    if (cancelRenderTask && *cancelRenderTask) {
+                        DBG("Cancelling rendering task!!!");
+                        turnOffAllPlugins (edit);
```

Comment:
Why is this here for cancelling? Could that cause issues if there's another render happening?

Reply:
are you asking re: turnoffallplugins? was doing it here because it was done in the case of successful render and thought that returns to same state in either case

Reply:
Yeah for turnOffAllPlugins. Maybe it's fine, but was slightly confused by it when canceling rendering.